src/components/AccountDrawer/HideSpamToggle.tsx

```diff
@@ -0,0 +1,19 @@
+import { t } from '@lingui/macro'
+import { useAtom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+import { SettingsToggle } from './SettingsToggle'
+
+export const hideSpamAtom = atomWithStorage<boolean>('hideSmallBalances', true)
```

Comment:
why do we use atomWithStorage for some persisted user settings & then redux store for others?

Reply:
agreed, why can't we put this in the `user` slice

Reply:
the settings components in the account drawer all use atoms for simplicity, imo writing all of the boilerplate reducer code just to flip a boolean is a worse solution